### PR TITLE
Qh dspdc 1536 data model exporter

### PR DESCRIPTION
**Pull request by eparkmonster**
Created skeleton for data model exporter 
[relevant ticket](https://broadinstitute.atlassian.net/browse/DSPDC-1536)

Questions regarding the skeleton:

- Verify if we want to CLI arguments to be: -f "filePath_name" -c "classPath_name"
- Naming convention for the script (currently called __init__.py)
- We want to hand these args off to 1537, that would take care of the RDF -> JSON
- Class file path name is verified in a separate function, that would be done in 1538

Just wanted some clarification if this was the scope of this ticket, and that 1537 and 1538 would build off of this to implement RDF ->JSON and class file parsing

### CODE REVIEW

File: tools/data-model-exporter/data_model_exporter/__init__.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
+__version__ = '0.1.0'
```
Comment (aherbst-broad) [inline tools/data-model-exporter/data_model_exporter/__init__.py:1]: I don't think you need this.

Comment (aherbst-broad) [inline tools/data-model-exporter/data_model_exporter/__init__.py:1]: Check the spec linked in the parent ticket, I believe there is a suggested file name there. This shouldn't be named __init__.py

File: tools/data-model-exporter/data_model_exporter/__init__.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+#!/usr/bin/env python
+
+# Create a basic CLI tool that parses the arg list defined in the spec
```
Comment (aherbst-broad) [inline tools/data-model-exporter/data_model_exporter/__init__.py:4]: This should be a docstring `"""` vs `#` (see https://www.python.org/dev/peps/pep-0257/if you're curious)

File: tools/data-model-exporter/data_model_exporter/__init__.py (hunk 1 above)
Comment (aherbst-broad) [reply tools/data-model-exporter/data_model_exporter/__init__.py:1]: Why the double `data_model_exporter` directory structure?

**Review by kreinold — COMMENTED** [review]
No suggestions for change from me.

File: tools/data-model-exporter/data_model_exporter/__init__.py (hunk 1 above)
Comment (eparkmonster) [reply tools/data-model-exporter/data_model_exporter/__init__.py]: when we created the directory using poetry it set it up in that way.
https://python-poetry.org/docs/basic-usage/

File: tools/data-model-exporter/data_model_exporter/dmExporter.py [hunk 3]
```diff
@@ -0,0 +1,31 @@
+"""!/usr/bin/env python"""
```
Comment (aherbst-broad) [inline tools/data-model-exporter/data_model_exporter/dmExporter.py:1]: Sorry, I meant, the comment below on lines 3-4 should be docstringed, not the shebang (this should _not_ be commented or docstringed)

**Pull request closed without merging**